### ISSUE TITLE
[BUG]: return_train_score not returning train score

### ISSUE THREAD

**Issue by jslomkowski** [pycaret/pycaret#2458]
### pycaret version checks

- [X] I have checked that this issue has not already been reported [here](https://github.com/pycaret/pycaret/issues).

- [X] I have confirmed this bug exists on the [latest version](https://github.com/pycaret/pycaret/releases) of pycaret.

- [ ] I have confirmed this bug exists on the main branch of pycaret.

### Issue Description

In create_model when cross_validation=False, return_train_score returns test performance and not train. This may be confusing.

### Reproducible Example

```python
import numpy as np
import pandas as pd
from pycaret.regression import *


def make_dummy_benchmark():
    X_train = pd.DataFrame(np.ones([10, 5]), columns=list('ABCDE'))
    y_train = pd.DataFrame(np.ones([10, 1]), columns=['target'])
    X_test = pd.DataFrame(np.ones([2, 5]), columns=list('ABCDE')) * 2
    y_test = pd.DataFrame(np.ones([2, 1]), columns=['target']) * 3
    df_train = pd.concat([X_train, y_train], axis=1)
    df_test = pd.concat([X_test, y_test], axis=1)
    return pd.concat([df_train, df_test], axis=0).reset_index(drop=True)


def train_test_split(df, col, split_value):
    df = df.sort_values(col)
    train = df[df[col] < split_value]
    test = df[df[col] >= split_value]
    return test, train


df = make_dummy_benchmark()
test, train = train_test_split(df, 'A', 2)

exp = setup(train, 'target', test_data=test, preprocess=False)

model = create_model('dt', cross_validation=False, return_train_score=True)
```

### Expected Behavior

When cross_validation=False, create_model should return train performance if return_train_score=True

### Actual Results

```python-traceback
run the code example
```

### Installed Versions

<details>
'2.3.10'
</details>

**Comment by jslomkowski** [comment]
Is anyone aware of this issue?

**Comment by ngupta23** [comment]
@jslomkowski Thanks for reporting this and for providing the reproducible example. I have been able to recreate this issue, Currently, we are working on the 3.0.0rc1 release so this is likely not going to go into that. We have marked it to be fixed in the 3.0.0 release. But if you are willing to submit a PR for this fix before that, we would be happy to accept it, If you do, please branch of `develop` and merge back into `develop`. 

Thanks!

**Comment by ngupta23** [comment]
Actually, this seems to be fixed in the `develop` branch so it should be released with the 3.0.0rc release.

Can you double check this by installing the expected RC release using the following (in a clean environment)

!pip install git+https://github.com/pycaret/pycaret.git@develop


![image](https://user-images.githubusercontent.com/33585645/167724688-66e524df-4189-4b4b-bf6b-fbee0ecaa788.png)

**Comment by moezali1** [comment]
I have tested and its working now.
